assets/js/blocks/cart/block.js

```diff
@@ -87,8 +90,10 @@ const Block = ( { attributes, children, scrollToTop } ) => (
 		<StoreNoticesContainer context="wc/cart" />
 		<SlotFillProvider>
 			<CartProvider>
```

Comment:
I'm wondering if we should combine Cart and CartEvents providers? At the moment, the Cart provider just passed the children down and has a redirectUrl

Reply:
Please see Mike's comment below